sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -37,6 +41,18 @@ public BatchedLogs(List<T> logsList, BinaryData logsData)
             public BinaryData LogsData { get; }
         }
 
+        internal readonly struct RunningTask<T>
```

Comment:
This is pretty cool - a lot cleaner than what I did with using Tuples in the SB snippet I linked.